gui/default/index.html

```diff
@@ -535,7 +535,7 @@ <h4 class="panel-title">
                           </span>
                           <span ng-if="folder.versioning.type != 'external'">
                             <span ng-if="(folder.versioning.type == 'trashcan' || folder.versioning.type == 'simple')" tooltip data-original-title="{{'Clean out after' | translate}}">
-                              &ensp;<span class="fa fa-calendar"></span>&nbsp;{{folder.versioning.params.cleanoutDays * 86400 | duration:"d"}}
+                              &ensp;<span class="fa fa-calendar"></span>&nbsp;<span ng-if="folder.versioning.params.cleanoutDays == 0" translate>Disabled</span><span ng-if="folder.versioning.params.cleanoutDays > 0">{{folder.versioning.params.cleanoutDays * 86400 | duration:"d"}}</span>
```

Comment:
This seems unrelated but actually it is an oversight of my much older PR to display versioning information in the GUI. It wasn't noticed then because the default cleanup interval was `0`, hence the information was hidden, but now it's visible, so it's necessary to show `Disabled` instead (as otherwise it displays `<1d`).